llama_stack/apis/common/type_system.py

```diff
@@ -77,7 +74,3 @@ class AgentTurnInputType(BaseModel):
     ],
     Field(discriminator="type"),
 ]
-
-ArrayType.model_rebuild()
```

Comment:
cc @ashwinb The recursive pydantic build here causes recursive loop with our generation script, I removed it for now since will need some time to see how to properly generate these types to OpenAPI, and we're going with the `ChatCompletionInputType`. WDYT?

Reply:
I see yeah can you comment it out instead and add this note there?